.github/ISSUE_TEMPLATE/issue.yaml

```diff
@@ -0,0 +1,96 @@
+name: New Issue
+description: Create a new issue
+body:
+- type: markdown
+  attributes:
+    value: |
+      NOTE: If your issue is a security concern, please send an email to security@gitea.io instead of opening a public issue.
+- type: markdown
+  attributes:
+    value: |
+      1. Please speak English, this is the language all maintainers can speak and write.
+      2. Please ask questions or configuration/deploy problems on our Discord
+         server (https://discord.gg/gitea) or forum (https://discourse.gitea.io).
+      3. Please take a moment to check that your issue doesn't already exist.
+      4. Make sure it's not mentioned in the FAQ (https://docs.gitea.io/en-us/faq)
+      5. Please give all relevant information below for bug reports, because
+         incomplete details will be handled as an invalid report.
+- type: input
+  id: gitea-ver
+  attributes:
+    label: Gitea Version
+    description: Gitea version (or commit reference) your instance is running
+    placeholder: v1.14.3
```

Comment:
Is there maybe a way so that the placeholder will also be used as default?
Because currently, it is only displayed when no text is present.
GitHub still requires you to fill that field before you can submit the issue.

Reply:
I can indeed specify a default value, however this would need to be updated every release.

Reply:
But the placeholder not?

Reply:
The placeholder could also be updated, however I consider placeholders as example values and not the value that a user should enter.